docs/content/concepts/automation/declarative-automation/customizing-automation-conditions.mdx

```diff
@@ -168,7 +165,7 @@ It's common to have use cases similar to pre-built policies but with minor diffe
 
 ### `AutomationCondition.eager()`: Ignoring missing dependencies
 
-By default, `AutomationCondition.eager()` will not materialize an asset partition if it has any missing dependencies. If it is expected to have missing upstream data, remove `~AutomationCondition.any_deps_missing()` from the eager policy to allow execution:
+By default, `AutomationCondition.eager()` will not materialize a target if it has any missing dependencies. If it is expected to have missing upstream data, remove `~AutomationCondition.any_deps_missing()` from the eager policy to allow execution:
```

Comment:
Explaining the reasoning behind this would be good.

"Missing dependencies" also is a little ambiguous here between the dependencies between "definitions" with the change to the word "target" The switch between "dependencies" and "upstream data" is also a little confusing. I think standardizing on "upstream data" will be better, since dependency has a double meaning between the definition graph and the graph of materializations.